vllm/config/kv_transfer.py

```diff
@@ -64,6 +64,11 @@ class KVTransferConfig:
     enable_permute_local_kv: bool = False
     """Experiment feature flag to enable HND to NHD KV Transfer"""
 
+    kv_load_retry_policy: Literal["recompute", "abort"] = "recompute"
```

Comment:
IMO we should reserve the "abort" term to mean client-aborted if possible. So could this be `"fail"` or `"fail_request"` instead?